src/main/java/com/minecolonies/coremod/client/gui/WindowHireWorker.java

```diff
@@ -334,4 +336,12 @@ protected String createColor(final Skill primary, final Skill secondary, final S
         }
         return "";
     }
+
+    private boolean studentHiring(ICitizenDataView citizen) {
```

Comment:
where is this called?

Reply:
Dangit. thought I'd removed that. Was using it to debug, then refactored the code it contained.